src/Nethermind/Nethermind.Synchronization/Nethermind.Synchronization.csproj

```diff
@@ -6,6 +6,9 @@
     
     
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    
+    
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
Does this allow this for the whole project or just `unsafe` blocks?

Reply:
Whole project but only used it for `[SkipLocalsInit]` in the method where I also called `.Clear` on the stackalloc'd span